test/integration/clusterio.js

```diff
@@ -158,7 +158,9 @@ describe("Integration of Clusterio", function() {
 					],
 					[
 						"allow_commands", true,
-						"allow-commands", "Allow Lua commands: Yes.",
+						"allow-commands", (process.platform === "darwin"
+							? "Yes"
+							: "Allow Lua commands: Yes."),
```

Comment:
Are you sure you're running these tests with the latest released version of Factorio? The output of this command recently changed and it makes no sense for the output to differ based on which platform Factorio was built for.

Reply:
I'm running in 1.1.19, though it was an MacOS stuff